Sources/Controllers/MessagesViewController.swift

```diff
@@ -36,10 +36,18 @@ UICollectionViewDelegateFlowLayout, UICollectionViewDataSource {
     /// The `InputBarAccessoryView` used as the `inputAccessoryView` in the view controller.
     open lazy var messageInputBar = InputBarAccessoryView()
 
+    /// A Boolean value that determines whether the `MessagesCollectionView` scrolls to the
+    /// last item whenever the `InputTextView` begins editing.
+    ///
+    /// The default value of this property is `false`.
+    /// NOTE: This calls scrollToLastItem where as the below flag calls scrollToBottome - check methods for differences
+    open var scrollsToLastItemOnKeyboardBeginsEditing: Bool = false
```

Comment:
I believe something similar was removed for auto-scroll on opening the controller - https://github.com/MessageKit/MessageKit/pull/395 - are we sure we want it back?

Reply:
If you look closer, on first layout was removed but on begin editing was still there https://github.com/MessageKit/MessageKit/pull/395/files#diff-5a6cda04708cf274c890a0278aaeade7L59

This happens after user taps on the input bar so shouldn't cause problems